guides/common/assembly_configuring-repositories.adoc

```diff
@@ -0,0 +1,4 @@
+[id="configuring-repositories_{context}"]
+= Configuring repositories
```

Comment:
Assemblies are currently only containers for includes, no content gets included (not even a heading). The first include in an assembly is usually a con_*.adoc that includes a heading and an assembly introduction.

Reply:
I introduced `guides/common/modules/con_configuring-repositories.adoc`. Is that what you intended?